ksqldb-functional-tests/src/test/java/io/confluent/ksql/test/planned/PlannedTestUtils.java

```diff
@@ -50,7 +50,7 @@ public static boolean isNotExcluded(final TestCase testCase) {
     // Place temporary logic here to exclude test cases based on feature flags, etc.
     return !(boolean) testCase
         .properties()
-        .getOrDefault(KsqlConfig.KSQL_CREATE_OR_REPLACE_ENABLED, false);
+        .getOrDefault(KsqlConfig.KSQL_KEY_FORMAT_ENABLED, false);
```

Comment:
`PlannedTestsUpToDateTest` currently fails because of two changes:
- the removal of the create or replace feature flag from this properties blacklist, as this adds a field into the plan
- the additional of the `DefaultFormatInjector` in this PR, which injects `KEY_FORMAT` into CS/CT statements where only `VALUE_FORMAT` has been explicitly provided, which is currently all tests.

I think these two changes are acceptable, and therefore the way to fix the test is to regenerate test plans, but I'm curious to hear what others think, especially regarding whether it makes sense to inject key/value formats into CS/CT statements that don't explicitly provide them. Note that this behavior is inline with the `DefaultSchemaInjector` injecting schemas into statements for which schema inference is performed.

Reply:
> I've leave the removal of the KSQL_CREATE_OR_REPLACE_ENABLED to another PR. It's not related to this PR.

Turns out the removal of this feature flag does not affect whether the existing historic plans are up to date or not, as the newly added `orReplace` flag is optional and inferred to be false if not present.

> DefaultFormatInjector should, (as I've noted above), do nothing if the feature flag is not set. Hence no plans should need to change. In general, if you find you need to generate a new set of plans before the feature flag is removed... something is wrong! ;)

See https://github.com/confluentinc/ksql/pull/6194#discussion_r489921795 for why the injector is still adding KEY_FORMAT even if the feature flag is not set.

In light of this, I've gone with the approach we discussed offline of updating `KsqlPlanV1` to not include the `statementText` in comparisons, as we decided this was the lesser evil compared to either generating new plans or rewriting the existing ones.